homeworks/magurmarian/hw01/calculation.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import sys
+
+from pyats.aetest import Testcase, test, main
+from somefunc import add, divide
+
+parameters = {
+    'num1': 3,
+    'num2': 0
+}
+
+
+class SmokeTest(Testcase):
+
+    @test
+    def test_add(self, steps, num1, num2):
+        with steps.start('Result bigger then 0') as step:
+            if add(num1, num2) > 0:
```

Comment:
Tests should be skipped if result is less than zero. In your case, zero is not included.
Also, steps comments could me more informative.